x/stakeibc/keeper/host_zone_test.go

```diff
@@ -248,4 +251,11 @@ func (s *KeeperTestSuite) TestAddDelegationToValidator() {
 	// Attempt to modify a validator that doesn't exist - it should fail
 	err = s.App.StakeibcKeeper.AddDelegationToValidator(s.Ctx, hostZone, "does_not_exist", sdkmath.NewInt(1000), "")
 	s.Require().ErrorContains(err, "validator not found")
+
+	// Attempt to subtract more than the total delegations on the host - it should fail
```

Comment:
Can you explain this test? My understanding is that `AddDelegationToValidator` is only called when new delegations are added to a validator